src/frontend/Player/i18n/plyrTranslation.ts

```diff
@@ -0,0 +1,214 @@
+import { defineMessages } from 'react-intl';
+
+export const i18nMessages = defineMessages({
+  2160: {
+    defaultMessage: '4K',
+    description: 'Label associatied to the 2160px quality badge',
+    id: 'plyr.controls.quality.badge.2160',
+  },
+  1440: {
+    defaultMessage: 'HD',
+    description: 'Label associatied to the 1440px quality badge',
+    id: 'plyr.controls.quality.badge.1440',
+  },
+  1080: {
+    defaultMessage: 'HD',
+    description: 'Label associatied to the 1080px quality badge',
+    id: 'plyr.controls.quality.badge.1080',
+  },
+  720: {
+    defaultMessage: 'HD',
+    description: 'Label associatied to the 720px quality badge',
+    id: 'plyr.controls.quality.badge.720',
+  },
+  576: {
+    defaultMessage: 'SD',
+    description: 'Label associatied to the 576px quality badge',
+    id: 'plyr.controls.quality.badge.576',
+  },
+  480: {
+    defaultMessage: 'SD',
+    description: 'Label associatied to the 480px quality badge',
+    id: 'plyr.controls.quality.badge.480',
+  },
+  advertisement: {
```

Comment:
Do we need to set all strings? I don't think we're ever using this one

Reply:
We have to redefine all `i18n` settings so I think translating everything can be done. If we don't want to translate them we can keep the original version.
Translation can also be used by other users.